### PR TITLE
improved ui.share action. Added delegate for share actions.

### PR DESCRIPTION
**Pull request by aimonx85**
Feel free to rearrange my code since those are my first lines in ObjC

### CODE REVIEW

File: ios/config.json [hunk 1]
```diff
@@ -22,6 +22,6 @@
 		"FacebookAppID": "$(facebook.facebookAppID)",
 		"FacebookDisplayName": "$(facebook.facebookDisplayName)",
 		"CFBundleURLTypes[has(CFBundleURLSchemes)].CFBundleURLSchemes[]": "fb$(facebook.facebookAppID)",
-		"LSApplicationQueriesSchemes[]": "fbauth"
+		"LSApplicationQueriesSchemes": ["fbapi","fbapi20130214", "fbapi20130410","fbapi20130702","fbapi20131010","fbapi20131219","fbapi20140410","fbapi20140116","fbapi20150313","fbapi20150629","fbauth","fbauth2","fb-messenger-api20140430"]
```
Comment (rampr) [inline ios/config.json:25]: Please help us understand what these schemes are for ?

Any reference we can look at ?

Comment (aimonx85) [inline ios/config.json:25]: look here: https://developers.facebook.com/docs/ios/ios9

Comment (rampr) [inline ios/config.json:25]: Thanks!

**Comment by jishnu7** [comment]
Thanks @aimonx85

**Pull request merged**